packages/nimble-components/src/spinner/tests/spinner.mdx

```diff
@@ -0,0 +1,45 @@
+import {
+    Canvas,
+    Meta,
+    Controls,
+    Stories,
+    Title,
+    Description
+} from '@storybook/blocks';
+import * as spinnerStories from './spinner.stories';
+
+<Meta of={spinnerStories} />
+
+<Title of={spinnerStories} />
+<Description of={spinnerStories} />
+
+<Canvas of={spinnerStories.spinner} />
+<Controls of={spinnerStories.spinner} />
+
+# Usage Docs
+
+<br />
+
+## Sizing
+
+The 64px spinner is a less commonly used size reserved for large negative empty space. Only one 64px spinner should be used on a single screen at one time.
+
+The 32px spinner is a more commonly used size for negative empty space. No more than three 32px spinners should be used on a single screen at one time.
+
+The 16px spinner is the most compact option for tight spaces such as grid cells and dense UI interfaces without much negative empty space.
+
+Multiple 16px spinners can be used on a single screen at one time.
+
+## Styling / Theme
+
+Use the `default` appearance Color UI version for backgrounds with color (e.g. purple, blue).
+
+Do not use the `accent` appearance on any colored backgrounds, instead use the `default` Color UI version.
```

Comment:
I think these two lines are essentially saying the same thing so could be collapsed to one paragraph.:
```suggestion
Use the `default` appearance in the Color UI theme for backgrounds with color (e.g. purple, blue). Do not use the `accent` appearance on any colored backgrounds.
```